### PR TITLE
14 update the component rendering to avoid their overlap in reduced screen size mode

### LINKED ISSUE #14: Update the component rendering to avoid their overlap in reduced screen size mode
- [ ] The plot component renders over the clear button when the screen size is reduced. 
- [ ] The menu Icon button in reduced size is more than the dashboard's header.

### PR DESCRIPTION
**Pull request by SutarPrasad**
- [x] Closes #14 

This PR involved adjusting the overlap of the plot with a clear button
Secondly, the radio MenuIcon in reduced view was not easily visible as its visualisation came out of the header background.
Radio buttons spacing is also adjusted
Y label didn't show the axis name completely in reduced size mode hence updated that too.

### CODE REVIEW

**Comment by SutarPrasad** [comment]
Once react version is upgraded to 18 these checks will resolve. PR 

- [ ] #12

**Comment by gareth-j** [comment]
For me this makes the select site text move to a weird location
select_site_text

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/styled": "^11.11.0",
```
Comment (gareth-j) [inline package.json:7]: Are we using emotion anywhere?

File: src/components/ObsBox/ObsBox.js [hunk 2]
```diff
@@ -55,7 +56,7 @@ class ObsBox extends React.Component {
         // We only set the title of the graph if there's one site selected
         let title = null;
         const xLabel = "Date";
-        const yLabel = `Concentration${units}`;
+        const yLabel = `Concentration <br>${units}`;
```
Comment (gareth-j) [inline src/components/ObsBox/ObsBox.js:59]: I'm not sure we want a `<br>` here?

**Comment by SutarPrasad** [comment]
> For me this makes the select site text move to a weird location select_site_text

Ignore this PR it will need to be updated at the end, once all other PR's are merged. Changing to draft.

File: src/components/ObsBox/ObsBox.js [hunk 3]
```diff
@@ -85,13 +86,14 @@ class ObsBox extends React.Component {
...
-      clearButton = <NiceButton onClick={this.props.clearSources}>Clear</NiceButton>;
+      clearButton = <NiceButton onClick={this.props.clearSources} >Clear</NiceButton>;
+      
     }
 
     const availableSpecies = Object.keys(this.props.processedData);
 
     return (
-      <div className={styles.container}>
+      <div className={styles.container} style={{display:'flex'}}>
```
Comment (gareth-j) [inline src/components/ObsBox/ObsBox.js:96]: Can the style parameters be added to the CSS instead?

File: src/components/RadioButtons/RadioButtons.js [hunk 4]
```diff
@@ -48,7 +48,7 @@ class RadioButtons extends React.Component {
     }
 
     return (
-      <div className={styles.buttons} onChange={this.onChangeValue}>
+      <div className={styles.buttons} onChange={this.onChangeValue} style={{display: "contents"}} >
```
Comment (gareth-j) [inline src/components/RadioButtons/RadioButtons.js:51]: Add the style parameters to the CSS file instead?

**Review by gareth-j — CHANGES_REQUESTED** [review]
Some questions and see screenshot added to the PR.

**Pull request closed without merging**